assignment 5/day 6/index.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs');
+const files = ["./files/mongodb.txt",
```

Comment:
In case of distributing elements into multiple lines, distribute the first element to the next line as well.